.github/workflows/test_develop.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   trigger_tests:
     strategy:
       matrix:
-        branch: [develop, release/cycle-3.119, release/cycle-3.120]  # List other branches here
+        branch: [develop, release/cycle-3.119, release/cycle-3.120, release/cycle-3.121]  # List other branches here
```

Comment:
I wonder if we could remove 119, it has been failing for a while anyway.
<img width="676" alt="Screenshot 2025-04-07 at 12 39 54" src="https://github.com/user-attachments/assets/1ea50b7f-82fd-440c-824f-d6d6566f8924" />

Reply:
Just seen that it was already suggested.